presentation/common/src/presentation-common/hierarchy/Key.ts

```diff
@@ -274,3 +278,23 @@ export interface LabelGroupingNodeKeyJSON extends GroupingNodeKeyJSON {
  * @public
  */
 export type NodeKeyJSON = BaseNodeKeyJSON | ECInstancesNodeKeyJSON | ECClassGroupingNodeKeyJSON | ECPropertyGroupingNodeKeyJSON | LabelGroupingNodeKeyJSON;
+
+/**
+ * Data structure that describes an instance keys select query
+ * @beta
+ */
+export interface InstanceKeysSelectQuery {
+  /** The query to select instance keys */
+  query: string;
+  /** The query parameters */
+  bindings: string[];
+}
+
+/**
+ * Serialized [[InstanceKeysSelectQuery]] JSON representation.
+ * @beta
+ */
+export interface InstanceKeysSelectQueryJSON {
```

Comment:
is there a reason to have this interface? member types completely match `InstanceKeysSelectQuery`..

Reply:
Removed.